### PR TITLE
[log-shipper] Documentation updates

### PR DESCRIPTION
**Pull request by Beastlex**
## Description
There is some misconception in the description. And clients can read as 'will not transform' instead of 'were not transformed' 

## Why do we need it, and what problem does it solve?
Exclude inaccuracies in the client's reading.

## Why do we need it in the patch release (if we do)?
Not necessary.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: log-shipper
type: chore
summary: Documentation updates.
impact_level: low
```

### CODE REVIEW

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 1]
```diff
@@ -74,16 +74,18 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
...
-Log'и до применения правил трансформации:
+С помощью такой команды вы сможете увидеть log'и до применения правил трансформации (`d8_cluster_source_*` - является
+первой стадией обработки согласно вывода команды `vector graph` ):
 
 ```bash
 vector tap d8_cluster_source_*
 ```
 
-Измененные логи:
+Измененные логи, которые поступают на вход следующих в цепочки компонентов каналов:
```
Comment (z9r5) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:88]:
```suggestion
Измененные логи, которые поступают на вход следующих в цепочке компонентов каналов:
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 2]
```diff
@@ -74,16 +74,18 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
 
 ### Получить не обработанные образцы log'ов
 
-Вы можете выполнить команду `vector tap` чтобы получить все необработанные образцы для всех каналов.
-Единственный аргумент, который принимает команда - ID стадии обработки (можно передавать в формате glob).
+Вы можете использовать команды `vector tap`, чтобы получить входные данные для любых компонентов каналов.
+Единственный аргумент, который нужно задать - ID стадии обработки (тажке поддерживаются выборки в формате glob.
+Например, `d8_cluster_source_*`).
```
Comment (z9r5) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:79]:
```suggestion
Вы можете использовать команду `vector tap`, чтобы получить входные данные для любых компонентов каналов.
Единственный аргумент, который нужно задать — ID стадии обработки. Также поддерживаются выборки в формате glob (например, `d8_cluster_source_*`).
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 3]
```diff
@@ -74,16 +74,18 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
...
 
-Log'и до применения правил трансформации:
+С помощью такой команды вы сможете увидеть log'и до применения правил трансформации (`d8_cluster_source_*` - является
+первой стадией обработки согласно вывода команды `vector graph` ):
```
Comment (z9r5) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:82]:
```suggestion
С помощью такой команды вы сможете увидеть log'и до применения правил трансформации (`d8_cluster_source_*` является первой стадией обработки согласно вывода команды `vector graph`):
```

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
@Beastlex Please fix the DCO

File: modules/460-log-shipper/docs/ADVANCED_USAGE.md [hunk 4]
```diff
@@ -72,22 +72,23 @@ Example of the output:
...
-The only argument to the command is the ID of the pipeline stage (glob patterns are allowed).
+You may use the `vector tap` command to get input data for any channel components.
+The only argument to be set is the ID of the processing stage. Patterns in a glob format are also supported for selections (e.g., `d8_cluster_source_*`).
 
-Logs before applying any transforms:
+By using the following command, you can see the log's before applying the transformation rules (`d8_cluster_source_*` is
+the first stage of processing according to the output of the `vector graph` command):
 
 ```bash
 vector tap d8_cluster_source_*
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE.md:82]:
```suggestion
vector tap 'cluster_logging_config/*'
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE.md [hunk 5]
```diff
@@ -72,22 +72,23 @@ Example of the output:
...
 ```
 
-Transformed logs:
+Transformed logs that are received at the input of the next channel components in the chain:
 
 ```bash
 vector tap d8_tf_*
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE.md:88]:
```suggestion
vector tap 'transform/*'
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 6]
```diff
@@ -74,16 +74,17 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
 
 ### Получить не обработанные образцы log'ов
 
-Вы можете выполнить команду `vector tap` чтобы получить все необработанные образцы для всех каналов.
-Единственный аргумент, который принимает команда - ID стадии обработки (можно передавать в формате glob).
+Вы можете использовать команды `vector tap`, чтобы получить входные данные для любых компонентов каналов.
+Единственный аргумент, который нужно задать - ID стадии обработки (тажке поддерживаются выборки в формате glob.
+Например, `d8_cluster_source_*`).
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:79]:
```suggestion
Например, `cluster_logging_config/*`).
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 7]
```diff
@@ -74,16 +74,17 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
...
 
-Log'и до применения правил трансформации:
+С помощью следующей команды вы сможете увидеть логи до применения правил трансформации (`d8_cluster_source_*` является первой стадией обработки согласно вывода команды `vector graph`):
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:81]:
```suggestion
С помощью следующей команды вы сможете увидеть логи до применения правил трансформации (`cluster_logging_config/*` является первой стадией обработки согласно вывода команды `vector graph`):
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 8]
```diff
@@ -74,16 +74,17 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
...
 
 ```bash
 vector tap d8_cluster_source_*
 ```
 
-Измененные логи:
+Измененные логи, которые поступают на вход следующих в цепочке компонентов каналов:
 
 ```bash
 vector tap d8_tf_*
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:90]:
```suggestion
vector tap 'transform/*'
```

File: modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md [hunk 9]
```diff
@@ -74,16 +74,17 @@ kubectl -n d8-log-shipper exec $pod -it -c vector -- bash
...
-Единственный аргумент, который принимает команда - ID стадии обработки (можно передавать в формате glob).
+Вы можете использовать команды `vector tap`, чтобы получить входные данные для любых компонентов каналов.
+Единственный аргумент, который нужно задать - ID стадии обработки (тажке поддерживаются выборки в формате glob.
+Например, `d8_cluster_source_*`).
 
-Log'и до применения правил трансформации:
+С помощью следующей команды вы сможете увидеть логи до применения правил трансформации (`d8_cluster_source_*` является первой стадией обработки согласно вывода команды `vector graph`):
 
 ```bash
 vector tap d8_cluster_source_*
```
Comment (nabokihms) [inline modules/460-log-shipper/docs/ADVANCED_USAGE_RU.md:84]:
```suggestion
vector tap 'cluster_logging_config/*'
```

**Review by nabokihms — CHANGES_REQUESTED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Backport failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/5658846459) for details.

**Pull request merged**